packages/sources/coinpaprika/src/endpoint/vwap.ts

```diff
@@ -0,0 +1,87 @@
+import { Requester, Validator } from '@chainlink/ea-bootstrap'
+import { ExecuteWithConfig, Config, InputParameters } from '@chainlink/types'
+import { NAME as AdapterName } from '../config'
+import { getCoinIds, getSymbolToId } from '../util'
+
+export const supportedEndpoints = ['vwap', 'crypto-vwap']
+
+export const endpointResultPaths = {
+  vwap: '0.price',
+  'crypto-vwap': '0.price',
+}
```

Comment:
Is this necessary if all result paths are the same?

Reply:
Currently, yes.